addons/mail/static/src/new/chat/chat_window.xml

```diff
@@ -48,8 +48,8 @@
                     <t t-if="thread.inCall">
                         <CallUI thread="thread" compact="true"/>
                     </t>
-                    <Thread id="thread.localId" t-key="thread.localId" messageHighlight="messageHighlight"/>
-                    <Composer composer="thread.composer" autofocus="props.chatWindow.autofocus" mode="'compact'" highlightReplyTo="messageHighlight.highlightMessage" dropzoneRef="contentRef"/>
+                    <Thread id="thread.localId" t-key="thread.localId" messageHighlight="messageHighlight" messageInEditId="state.messageInEditId" resetMessageInEdit="resetMessageInEdit.bind(this)"/>
+                    <Composer composer="thread.composer" autofocus="props.chatWindow.autofocus" mode="'compact'" highlightReplyTo="messageHighlight.highlightMessage" startEditingLastMessageOfCurrentUser="startEditingLastMessageOfCurrentUser.bind(this)" dropzoneRef="contentRef"/>
```

Comment:
I would be happy if we could find a shorter name for this feature. `editLastMessage`? Because `OfTheCurrent` user seams obvious to me.
You can also do `myProps="() => this.myFunction()"` instead of using `.bind(this)`. It doesn't really matter I guess, but the arrow function is documented in Owl.